src/reason-parser/reason_syntax_util.ml

```diff
@@ -393,10 +393,12 @@ let identifier_mapper f super =
 (** escape_stars_slashes_mapper escapes all stars and slases in an AST *)
 let escape_stars_slashes_mapper =
   let escape_stars_slashes str =
-    if String.contains str '/' then
+    if (String.contains str '/') ||
+       ((String.contains str '=') && (String.contains str '>')) then
```

Comment:
maybe this can be a simple `str = "=>"` since we're special casing the fat arrow. let me know

Reply:
This probably should be `str = "=>"` or something equivalent to avoid catching operators which don't need escaping like `>=` or `==>`.

Reply:
@hcarty that a better reason to do what I was suggesting, will fix.